jetty-websocket/websocket-api/src/main/java/org/eclipse/jetty/websocket/api/RemoteEndpoint.java

```diff
@@ -141,6 +141,24 @@
      */
     void setBatchMode(BatchMode mode);
 
+    /**
+     * Set the maximum number of frames which allowed to be waiting to be sent at any one time.
+     * The default value is -1, this indicates there is no limit on how many frames can be
+     * queued to be sent by the implementation.
```

Comment:
Please add some text that explains what happens if the limit is exceeded.
What operation will get what exception, etc.